client/src/app/pages/tasks/TaskDetails.tsx

```diff
@@ -2,27 +2,54 @@ import React from "react";
 import { useParams } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 
-import { Paths, TaskDetailsAttachmentRoute } from "@app/Paths";
+import { Paths, TaskDetailsAttachmentRoute, TaskFromApp } from "@app/Paths";
 import "@app/components/simple-document-viewer/SimpleDocumentViewer.css";
 import { formatPath } from "@app/utils/utils";
 import { TaskDetailsBase } from "./TaskDetailsBase";
+import { getTaskById } from "@app/api/rest";
+import { useState, useEffect } from "react";
 
 export const TaskDetails = () => {
   const { t } = useTranslation();
   const { taskId, attachmentId } = useParams<TaskDetailsAttachmentRoute>();
+  const { isFApplication } = useParams<TaskFromApp>();
+  const result = isFApplication === "true" ? true : false;
+  const [applicationName, setApplicationName] = useState<string | undefined>();
+  const [applicationId, setApplicationId] = useState<number | undefined>();
+
+  useEffect(() => {
```

Comment:
@rszwajko Can you have a look at these changes?  I don't remember exactly how these components hang together.

Reply:
I missed this comment. I will take a look.